### DISCUSSION TITLE
Please support flutter version

### DISCUSSION THREAD

**Discussion by aaazxxaaa** [ant-design/ant-design-mobile#6634] [Q&A]
有没有可能用flutter把antd mobile在实现一把, 那就太棒了, 或者在ant-design 下面在新建一个flutter版本的工程, 由社区来维护, 太需要了

**Comment by afc163** [comment]
https://github.com/pd4d10/ant-design-mobile-flutter

有过，不过很久没维护了。

**Reply by aaazxxaaa** [reply]
常年没维护，不敢用啊, 能否跟antd-rn版本一样, 在这个team下, 新建一个flutter版本的工程, 由社区来迭代

**Reply by afc163** [reply]
这不是仓库建在哪的问题，没有人使用和维护的话，你期望谁来写这个代码呢？

**Reply by abjbjkxfxfs** [reply]
我这边看到腾讯有一个flutter版本， 但是组件没antd mobile丰富
https://tdesign.tencent.com/flutter/overview
建议：因为蚂蚁这边的仓库流量比较大，完全可以开一个flutter 版本工程， 工程下面放一个维护群，招募维护人员， 很多人都愿意加入进来的，完全有希望搞起来的

**Reply by afc163** [reply]
我们现在没有会写 flutter 的。

**Reply by abjbjkxfxfs** [reply]
哈哈哈，你这是纯技术思维，就好比马总会写代码嘛？会技术嘛？不会的，但是他能网罗国内第一梯队的码农给他写代码，最终能把事儿干成，换个运营的视角来看，有平台，有流量，有人气，干成的希望很大啊

**Comment by zombieJ** [comment]
官网做 React 的，其他的都是社区自发的。可以再找找

**Reply by aaazxxaaa** [reply]
知道是做react的, 能不能像react native那种模式一样, 在这个team下面, 再开一个flutter版本的工程， 给社区来进行迭代
